scripts/generate_reference.py

```diff
@@ -1,67 +1,35 @@
 import argparse
-from pathlib import Path
 
 import antares_test_utils as antares_utils
+from run_command_building import *
 import sys
 
+# Parsing args
 parser = argparse.ArgumentParser()
-parser.add_argument("root", help="Directory containing studies",
+parser.add_argument("batch_name", help="Btch directory (containing studies)",
                     type=str)
-
-parser.add_argument("solver", help="Path to antares-solver",
+parser.add_argument("path_where_to_find_exe", help="Path to binary to exe",
```

Comment:
the option name ans it's description are not consistent

Reply:
corrected